app/Resources/FOSUserBundle/serializer/Model.User.yml

```diff
@@ -0,0 +1,11 @@
+FOS\UserBundle\Model\User:
```

Comment:
Ahh... got you wrong ;) Feel free to move but I guess this will not change anything particular, only one more folder to check :)